help.c

```diff
@@ -32,15 +30,21 @@
 #include <wchar.h>
 #include "globals.h"
 #include "keymap.h"
-#include "keymap_defs.h"
 #include "lib/lib.h"
 #include "mapping.h"
 #include "mbyte.h"
 #include "mutt_curses.h"
+#include "opcodes.h"
 #include "options.h"
 #include "pager.h"
 #include "protos.h"
 
+static const char *HelpStrings[] = {
+#define DEFINE_HELP_MESSAGE(opcode, help_string) [opcode] = help_string,
+  OPS(DEFINE_HELP_MESSAGE)
+  [OP_MAX] = NULL,
+};
```

Comment:
Okay for undefs. For the explicit array index, it's just an artifact of the way I thought about it I guess, no real reasons to have an explicit index here.